### PR TITLE
Fix unreachable code in Database/drivers/pdo/subdrivers

### PR DESCRIPTION
**Pull request by dev-ggallon**
Hello,
Fix unreachable code in the function `_list_tables` for this files :
- system/database/drivers/pdo/subdrivers/pdo_dblib_driver.php
- system/database/drivers/pdo/subdrivers/pdo_sqlsrv_driver.php

### CODE REVIEW

File: system/database/drivers/pdo/subdrivers/pdo_dblib_driver.php [hunk 1]
```diff
@@ -154,17 +154,17 @@ public function db_connect($persistent = FALSE)
 	 */
 	protected function _list_tables($prefix_limit = FALSE)
 	{
-		return 'SELECT '.$this->escape_identifiers('name')
-			.' FROM '.$this->escape_identifiers('sysobjects')
-			.' WHERE '.$this->escape_identifiers('type')." = 'U'";
+		$sql = 'SELECT '.$this->escape_identifiers('name')
+			  .' FROM '.$this->escape_identifiers('sysobjects')
```
Comment (narfbg) [inline system/database/drivers/pdo/subdrivers/pdo_dblib_driver.php]: Don't add more indentation please ... the only changes necessary are switching from `return` to `$sql =` above, please revert everything else.

**Comment by dev-ggallon** [comment]
I reversed the indentation and add new commit to remove unnecessary return

**Comment by narfbg** [comment]
Um ... no, that's even worse.

Revert it to *only* replacing `return` with `$sql =` in the PDO subdrivers.

**Comment by dev-ggallon** [comment]
OK, I removed the last commits, maybe that I can propose a new commit for the delete the unnecessary returns or standardize with this returns ? Because in each driver the code is different for the same function... ditto for the encoding for example: `$sql .=` or `$sql.` there are no rules when files are compared

File: system/database/drivers/pdo/subdrivers/pdo_dblib_driver.php [hunk 2]
```diff
@@ -164,7 +164,7 @@ protected function _list_tables($prefix_limit = FALSE)
 				.sprintf($this->_like_escape_str, $this->_like_escape_chr);
 		}
 
-		return $sql.' ORDER BY '.$this->escape_identifiers('name');
+		return $sql .' ORDER BY '.$this->escape_identifiers('name');
```
Comment (narfbg) [inline system/database/drivers/pdo/subdrivers/pdo_dblib_driver.php]: ... remove the space.

File: system/database/drivers/pdo/subdrivers/pdo_sqlsrv_driver.php [hunk 3]
```diff
@@ -193,7 +193,7 @@ protected function _list_tables($prefix_limit = FALSE)
 				.sprintf($this->_like_escape_str, $this->_like_escape_chr);
 		}
 
-		return $sql.' ORDER BY '.$this->escape_identifiers('name');
+		return $sql .' ORDER BY '.$this->escape_identifiers('name');
```
Comment (narfbg) [inline system/database/drivers/pdo/subdrivers/pdo_sqlsrv_driver.php]: ... here too.

**Comment by narfbg** [comment]
They are not unnecessary returns. On the contrary - what you propose is unnecessary assignments.

**Comment by dev-ggallon** [comment]
I deleted spaces... Ok for the unnecessary assignments, then it lacks in these three drivers:
- pdo_informix_driver
- pdo_ibm_driver
- pdo_4d_driver

sorry to insist, but I am a little bit manic sometimes;)

**Pull request merged**